### ISSUE TITLE
OS X support

### ISSUE THREAD

**Issue by mehdideveloper** [jocover/Simulink-MATLAB-HackRF#2]
Hi
I have installed Matlab R2015a on OS X 10.11 and it works with RTL-SDR.
How can I build and use this repo for Mac? Is it possible?

Thanks

**Comment by jocover** [comment]
Build hackrf lib you can see
https://github.com/mossmann/hackrf/tree/master/host

Get the Simulink-MATLAB-HackRF source from the GitHub project page

$ git clone git://github.com/jocover/Simulink-MATLAB-HackRF.git

Run MATLAB and setup the MEX compiler

        mex -setup

Switch to your Simulink-MATLAB-HackRF directory.

Start the build process (in MATLAB)

        make.m
Add the bin and the blockset directory to the MATLAB path environment.

**Comment by mehdideveloper** [comment]
It gives error. Here's what I did step by step:
1- I had compiled HackRF libraries and tools before, so didn't do anything for this.
2- Used "mex -setup" in Matlab to select C++ as the language.
3- Navigated to the Simulink-MATLAB-HackRF directory, right-clicked on "make.m" and selected "Run"

It gives 9 warnings and 1 error.
The error is:
 
```
/Users/mehdi/Documents/Radio_Electronics/Source/Simulink-MATLAB-HackRF/src/hackrf_source.cpp:284:3: error:
expected expression
                ssSetErrorStatusf( S, "No HackRF boards found.\n" );
                ^
/Users/mehdi/Documents/Radio_Electronics/Source/Simulink-MATLAB-HackRF/src/utilities.h:31:47: note: expanded from
macro 'ssSetErrorStatusf'
    sprintf(errorMessage, message, __VA_ARGS__); \
                                              ^
9 warnings and 1 error generated.


Error in make (line 67)
mex(options{:},'-outdir',HACKRF_BIN_DIR,'src/hackrf_source.cpp')
 ```

Matlab R2015a
OS X 10.11
(I have "hackrf_*" binaries in my PATH, also its libraries and header files are in '/usr/local/lib' and '/usr/local/include' directories)

**Comment by mehdideveloper** [comment]
OK, I solved the problem.
I just replaced this line 
`ssSetErrorStatusf( S, "No HackRF boards found.\n" );`
with the actual macro code:
```
    char errorMessage[512];
    sprintf(errorMessage, "No HackRF boards found.\n"); \
    ssSetErrorStatus(S, errorMessage);
```
In two files: hackrf_sink.cpp and hackrf_source.cpp
It compiled and I added 'bin' and 'blockset' to Matlab's PATH. 
I even executed the 'hackrf_mscript_demo.m' successfully.
But, where are the Simulink blocks?! I can't find HackRF blocks in Simulink library.

Thanks
Mehdi
